### PR TITLE
fix link to GSuite in application-access/controls

### PR DESCRIPTION
**Pull request by ashkulz**

### CODE REVIEW

File: docs/pages/application-access/controls.mdx [hunk 1]
```diff
@@ -79,6 +79,6 @@ allow:
   - [OIDC](../enterprise/sso/oidc.mdx)
   - [ADFS](../enterprise/sso/adfs.mdx)
   - [Azure AD](../enterprise/sso/azuread.mdx)
-  - [GSuite](../enterprise/sso/azuread.mdx)
+  - [GSuite](../enterprise/sso/google-workspace.mdx)
```
Comment (benarent) [inline docs/pages/application-access/controls.mdx:82]:
```suggestion
  - [Google Workspace](../enterprise/sso/google-workspace.mdx)
```
Updated to reflect G Workspace re-branding

**Review by russjones — DISMISSED** [review]

**Review by r0mant — DISMISSED** [review]

**Comment by ashkulz** [comment]
Not sure why "Checking reviewers" is failing? :man_shrugging:

**Comment by russjones** [comment]
@quinqu Can you take a look at this PR. I think the dismiss workflow should re-request reviews as well. Right now it shows up as just comments. What do you think?

**Review by russjones — APPROVED** [review]

**Comment by quinqu** [comment]
@russjones Yes, I can add that functionality.

**Comment by ashkulz** [comment]
Anything needed from my side to get this merged? I'd have expected such a small typo to be merged more quickly :slightly_smiling_face:

**Comment by ptgott** [comment]
Looks like this one fell through the cracks. Sorry about that! Closing because this is superseded by the more recent #9825 .

**Pull request closed without merging**